app/views/users/show.html.erb

```diff
@@ -1,12 +1,20 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Posts</title>
-  </head>
-  <body>
-    <h1> Selected user from the list.</h1>
-  </body>
-</html>
\ No newline at end of file
+<div id="user-container">
+  <%= render partial: "users/user_info", locals: { user: @user } %>
+</div>
+
+<div id="user-biography">
+  <h3>Biography</h3>
+  <p><%= @user.bio %></p>
+</div>
+
+<div id="post-container">
+  <% @user.recent_posts.each_with_index do |post, i| %>
+    <li>
+      <%= link_to "/users/#{@user.id}/posts/#{post.id}", id: "post" do %>
+        <%= render partial: "posts/post_info", locals: { post: post } %>
+      <% end %>
+    </li>
+  <% end %>
+</div>
+
+<%= link_to "See all posts", "/users/#{@user.id}/posts?page=1", class: "see-all-posts-button" %>
```

Comment:
- [ ] Great job with the functionalities @pushkar1995, Thou I notice clicking on the `See All` button keeps throwing an error. Also, for the second user, I can't see the `See All` button does s not throw w the error, and that is because we don't have a post there, adding one I noticed ice it also throws the error. Please address this issue before the next review request. 
### The issue

Uploading Screen Recording 2023-11-10 at 3.25.40 in the afternoon.mov…

Reply:
@TadesseDev I fixed the See all Posts button issue as mentioned by the reviewer without using paginate. Previously I had used the paginate but now I removed the paginate method and using different approach for the See All Posts Button.